docs/provider/ngrok.md

```diff
@@ -0,0 +1,21 @@
+## ngrok
+
+External Secrets Operator integrates with [ngrok](https://ngrok.com/) to sync Kubernetes secrets with [ngrok Secrets for Traffic Policy](https://ngrok.com/blog-post/secrets-for-traffic-policy).
+Only pushing secrets is supported. For security reasons, retrieving the secret value is not supported at this time.
```

Comment:
> For security reasons, retrieving the secret value is not supported at this time.

Could you elaborate a bit on this?

Reply:
I believe this was an initial design decision to avoid accidental leakage by anyone else on the account. With the current product, ngrok secrets are only able to be used inside of [traffic policy](https://ngrok.com/blog-post/secrets-for-traffic-policy) YAML and aren't meant as a general purpose secret solution at this time. They operate very similar to GitHub Actions secrets.